### PR TITLE
[jsrsasign] - interfaces fix of X509 and CSRUtil

### PR DESCRIPTION
**Pull request by kratklu1**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/jsrsasign/modules/KJUR/asn1/csr/CSRUtil.d.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ declare namespace jsrsasign.KJUR.asn1.csr {
     }
 
     interface ParamResponse {
-        subject: { array?: IdentityResponse };
+        subject: { array?: Array<[{ type: string; value: string; ds: string }]>, str?: string };
```
Comment (ffflorian) [inline types/jsrsasign/modules/KJUR/asn1/csr/CSRUtil.d.ts:14]: A single-value array in an array - is that correct?

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## jsrsasign ([<kbd>unpkg</kbd>](https://unpkg.com/browse/jsrsasign@latest/))
was missing the following properties:
1. SecureRandom
1. rng_seed_time
1. ECDSA
1. DSA
1. Signature

<details>
<summary>as well as these 14 other properties...</summary>
<p>MessageDigest, Mac, Cipher, ASN1HEX, CryptoJS, hex2b64, ipv6tohex, hextoipv6, hextoip, iptohex, crypto, asn1, jws, lang</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 5670710a06faec0746e1d64b2cf7b5df826e87ba
</p>

File: types/jsrsasign/modules/KJUR/asn1/csr/CSRUtil.d.ts (hunk 1 above)
Comment (kratklu1) [reply types/jsrsasign/modules/KJUR/asn1/csr/CSRUtil.d.ts:14]: unfortunately yes, that way it is written, see https://github.com/kjur/jsrsasign/blob/1536f252d70d25db7ade80b31f57be26e52b26f9/src/asn1csr-1.0.js#L412

Comment (ffflorian) [reply types/jsrsasign/modules/KJUR/asn1/csr/CSRUtil.d.ts:14]: Alright, thanks!

**Review by ffflorian — APPROVED** [review]

**Comment by kratklu1** [comment]
Ready to merge

**Pull request merged**